plugin/src/test/java/org/opensearch/ml/rest/RestMLRemoteInferenceIT.java

```diff
@@ -76,39 +77,39 @@ public void setup() throws IOException, InterruptedException {
         Thread.sleep(20000);
     }
 
-    public void testCreateConnector() throws IOException {
-        Response response = createConnector(completionModelConnectorEntity);
-        Map responseMap = parseResponseToMap(response);
-        assertNotNull((String) responseMap.get("connector_id"));
-    }
-
-    public void testGetConnector() throws IOException {
+    public void testCreate_Get_DeleteConnector() throws IOException {
```

Comment:
If any assertion fails, it will indicate which assertion is failing within the method, correct? I’ve noticed a lot of redundant resource creation in this integration test class, which I plan to remove step by step. In this case, we are now creating the connector resource once and reusing it for three tests, rather than creating the resource three separate times.